lib/sqlalchemy/sql/schema.py

```diff
@@ -1065,7 +1065,7 @@ def __init__(self, *args, **kwargs):
           :class:`_schema.Table`.  This is to support convenient
           usage within the :mod:`~sqlalchemy.ext.declarative` extension.
 
-        :param type\_: The column's type, indicated using an instance which
```

Comment:
the \_ is an escape so that "type_" is not seen as an internal hyperlink target.

see https://docutils.sourceforge.io/docs/user/rst/quickref.html#internal-hyperlink-targets

and https://docutils.sourceforge.io/docs/user/rst/quickref.html#escaping

an exact example of `name\_` is at: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#hyperlink-targets

Reply:
I edited the comment above to put the `\_` in backticks so it would render correctly.